cypress/integration/home.spec.ts

```diff
@@ -6,4 +34,15 @@ it('should have a working link to comparator page', () => {
   cy.url().should('equal', 'http://localhost:3000/comparator')
 })
 
+// Ask Belco how to get the footer with .findByRole, I can't get it working with footer or with contentinfo.
+
+it('should display footer text', () => {
+  cy.visit('/')
+
+  cy.contains('footer', 'Created with')
+  cy.contains('footer', 'by Mario')
```

Comment:
```ts
  cy.contains('footer', 'Created with')
  cy.contains('footer', 'by Mario')
```

How should I change this so that it uses `cy.findByRole`?
I tried `cy.findByRole('footer')` but that doesn't work... 😄

Reply:
I highly recommend using https://testing-playground.com/ to find out what queries are available to get something on an existing website.  You can install the Chrome/Firefox extension so then you can inspect https://octoclairvoyant.vercel.app/ like this:

<img width="1904" alt="CleanShot 2021-09-13 at 17 12 45@2x" src="https://user-images.githubusercontent.com/2677072/133109873-471868ba-a221-4c2f-9011-b9b84ed67a78.png">

It seems there is no accessible way to check the footer by its role, so we would have to stick to the current `cy.contains` here.

Reply:
That plugin is really cool! 😄